### PR TITLE
refactor(docs/help): add github link and create new help section

### PR DESCRIPTION
**Pull request by mohamedMok**
## I have read [the contributing guidelines](https://mozaic.adeo.cloud/Contributing/)

- [x] Yes
- [ ] No

## Does this PR introduce a [breaking change](https://mozaic.adeo.cloud/Contributing/Developers/GitConventions/#breaking-changes-)?

- [ ] Yes
- [x] No

## Describe the changes

GitHub issue number or Jira issue URL: https://jira.adeo.com/browse/MZC-68

### CODE REVIEW

File: src/docs/Help/WantToHelp/index.mdx [hunk 1]
```diff
@@ -1,21 +1,15 @@
 ---
```
Comment (tiloyi) [inline src/docs/Help/WantToHelp/index.mdx:1]: Hi @caqueste , Hi @mohamedMok ,
I wonder if this page should not be deleted altogether? Indeed, it is updated very rarely, is it useful to keep it?

File: src/docs/Help/Help/index.mdx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+## 2 - Via Github
+
+There’s another way, in case you don’t have access to slack, we can use [GitHub issues](https://github.com/adeo/mozaic-design-system/issues) as well. We might take a few hours to check that. Truth is, we’re not that used to receive GitHub issues. But as far as this gets popular, our response time will gain speed proportionally.
+_There’s just one rule: Questions should be posted in English._
+
+## 3 - Via e-mail
+
+Old but gold, e-mails never fail. But it’s true that it might take more time for us to answer via this channel. So we’d prefer to recommend you using either Slack or Github for proper and fast support.
+Currently, we support English, French, and Portuguese by e-mail 🙃 ! : tiago.rosa (at) adeo (dot) com
```
Comment (tiloyi) [inline src/docs/Help/Help/index.mdx:21]: ~~tiago.rosa (at) adeo (dot) com~~ => I think we have to change the email adresse of Tiago :(

File: src/docs/Help/WantToHelp/index.mdx (hunk 1 above)
Comment (caqueste) [reply src/docs/Help/WantToHelp/index.mdx:1]: @mohamedMok @tiloyi I agree, we can delete this page, I think it's useless 😉

File: packages/gatsby-theme-styleguide/src/gatsby-components/Menu/Menu.js [hunk 3]
```diff
@@ -208,7 +222,19 @@ class Menu extends Component {
                 />
                 <NavContainer>
                   {this.buildMenu(this.state.menuArray, true)}
-
+                  <ListItem>
+                    <MenuItemContainer>
```
Comment (caqueste) [inline packages/gatsby-theme-styleguide/src/gatsby-components/Menu/Menu.js:226]: @mohamedMok Currently, just the text and the icon are clickable. Can you make clickable the tile, please?

File: packages/gatsby-theme-styleguide/src/gatsby-components/Menu/Menu.js [hunk 4]
```diff
@@ -208,7 +222,19 @@ class Menu extends Component {
...
+                      <GithubLink
+                        href="https://github.com/adeo/mozaic-design-system"
+                        target="_blank"
+                      >
+                        Github
```
Comment (caqueste) [inline packages/gatsby-theme-styleguide/src/gatsby-components/Menu/Menu.js:231]: ~~Github~~GitHub

**Review by caqueste — CHANGES_REQUESTED** [review]

**Review by caqueste — APPROVED** [review]

**Pull request merged**